src/T2DMap.cpp

```diff
@@ -2594,20 +2594,19 @@ void T2DMap::mousePressEvent(QMouseEvent* event)
 
             TRoom* room = mpMap->mpRoomDB->getRoom(mCustomLinesRoomFrom);
             if (room) {
-                QAction* action = new QAction("undo", this);
+                QAction* action = new QAction(tr("Undo", "Menu option to undo the last point in the mapper"), this);
                 action->setToolTip(tr("Undo last point"));
                 if (room->customLines.value(mCustomLinesRoomExit).count() > 1) {
                     connect(action, &QAction::triggered, this, &T2DMap::slot_undoCustomLineLastPoint);
                 } else {
                     action->setEnabled(false);
                 }
 
-                QAction* action2 = new QAction("properties", this);
-                action2->setText("properties...");
+                QAction* action2 = new QAction(tr("Properties...", "Menu option to change properties of a line in the mapper"), this);
```

Comment:
`"Menu option to change the properties of a new custom exit line being added to a room in the mapper"`?

Reply:
Not sure about lengthening all these comments *for translation*, they aren't even shown to users.

Do you have any idea, why the action text was set without 3 dots and then changed in the following line immediately?

Reply:
Because when it is set in the constructor it silently gets copied into other places as well! (the tool-tip and/or the object name IIRC) whereas I only wanted the ellipses in the *displayed* text...

Reply:
Fair enough, I'll add that back in then and add your comment as comment why

Reply:
From the Qt Documentation:
> `QAction::QAction(const QString &text, QObject *parent = nullptr)`
> Constructs an action with some text and parent. If `parent` is an action group the action will be automatically inserted into the group.
> 
> The action uses a stripped version of text (e.g. `"&Menu Option..."` becomes `"Menu Option"`) as descriptive text for tool buttons. You can override this by setting a specific description with `setText()`. The same text will be used for tooltips unless you specify a different text using `setToolTip()`.